static/app/views/performance/browser/webVitals/webVitalsDetailPanel.tsx

```diff
@@ -181,6 +176,11 @@ export function WebVitalsDetailPanel({
         </Tooltip>
       );
     }
+    if (col.key === 'count') {
+      if (webVital === 'inp') {
+        return <AlignRight>{`Interactions`}</AlignRight>;
```

Comment:
```suggestion
        return <AlignRight>{t('Interactions')}</AlignRight>;
```
possibly? although the other column names won't be getting i18n. This is up to you